tests/console/ansible.pm

```diff
@@ -80,7 +94,7 @@ sub run {
     # Check Ansible version
     record_info('ansible --version', script_output('ansible --version'));
 
-    my $hostname = script_output 'hostname';
+    my $hostname = script_output 'hostname -s';
```

Comment:
Do you expect a different hostname than `localhost`?

Reply:
Yes, on ALP.

Reply:
It shows `localhost` https://pdostal-server.suse.cz/tests/2811#step/ansible/123

Reply:
Yes, `hostname -s` shows `localhost`.

I'm sorry I don't remember where and what was the error I got - but I did get an error where `hostname` didn't return `localhost`.

P.s: Ah yes, I know what was the error, for `hostname` I got `localhost.localdomain` … but I still don't remember where :(

Reply:
See for example openSUSE microOS: https://pdostal-server.suse.cz/tests/2856#step/ansible/118

Reply:
I have just checked that `hostname` is required by `yast2` and `net-tools` so just wondering whether it might not be a problem in the future.
Mainly when this can be solved by a regex the in `validate_script_ouput`